tests/equisolve_tests/numpy/models/test_linear_model.py

```diff
@@ -65,19 +77,16 @@ def equisolve_solver_from_numpy_arrays(self, X_arr, y_arr, alpha_arr, sw_arr=Non
         clf.fit(X=X, y=y, alpha=alpha, sample_weight=sw)
         return clf
 
-    def numpy_solver(self, X, y, sample_weights, regularizations):
-        _, num_properties = X.shape
-
-        # Convert problem with regularization term into an equivalent
-        # problem without the regularization term
-        regularization_all = np.hstack((sample_weights, regularizations))
-        regularization_eff = np.diag(np.sqrt(regularization_all))
-        X_eff = regularization_eff @ np.vstack((X, np.eye(num_properties)))
-        y_eff = regularization_eff @ np.hstack((y, np.zeros(num_properties)))
-        least_squares_output = np.linalg.lstsq(X_eff, y_eff, rcond=1e-13)
-        w_solver = least_squares_output[0]
-
-        return w_solver
+    num_properties = np.array([91, 100, 119, 221, 256])
+    num_targets = np.array([1000, 2187])
+    means = np.array([-0.5, 0, 0.1])
+    regularizations = np.geomspace(1e-5, 1e5, 3)
+    """
+    For the tests using the paramaters above the number of properties always
+    needs to be less than the number of targets.
+    Otherwise, the property matrix will become singualar,
+    and the solution of the least squares problem would not be unique.
+    """
```

Comment:
I think a pure comment is better no?
```suggestion
    # For the tests using the paramaters above the number of properties always
    # needs to be less than the number of targets.
    # Otherwise, the property matrix will become singualar,
    # and the solution of the least squares problem would not be unique.
```